docker-compose.yml

```diff
@@ -1,80 +1,74 @@
-      - KAFKA_ZOOKEEPER_TLS_TRUSTSTORE_PASSWORD=${ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD}
-      - KAFKA_ZOOKEEPER_TLS_VERIFY_HOSTNAME=false
-      - KAFKA_CFG_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM=
-      - KAFKA_CLIENT_USER=${KAFKA_CLIENT_USER}
-      - KAFKA_CLIENT_PASSWORD=${KAFKA_CLIENT_PASSWORD}
-      - KAFKA_CERTIFICATE_PASSWORD=${KAFKA_CERTIFICATE_PASSWORD}
-      - KAFKA_INTER_BROKER_LISTENER_NAME=INTERNAL
-      - KAFKA_CFG_SASL_ENABLED_MECHANISMS=PLAIN
-      - KAFKA_CFG_SASL_MECHANISM_INTER_BROKER_PROTOCOL=PLAIN
-      - KAFKA_SSL_CLIENT_AUTH=required
-    networks:
-      - tombolo-network
-    volumes:
-      - ${KAFKA_ZOOKEEPER_CERTS_PATH}:/bitnami/kafka/config/certs
+  # kafka:
+  #   hostname: ${KAFKA_HOST_NAME}
+  #   image: 'bitnami/kafka:latest'
+  #   ports:
+  #     - ${KAFKA_PORT}:${KAFKA_PORT}
+  #     - ${KAFKA_EXTERNAL_PORT}:${KAFKA_EXTERNAL_PORT}
+  #   environment:
+  #     - KAFKA_CFG_ZOOKEEPER_CONNECT=zookeeper:3181
+  #     - KAFKA_CFG_LISTENERS=INTERNAL://kafka:${KAFKA_PORT},CLIENT://kafka:${KAFKA_CLIENT_PORT},EXTERNAL://:${KAFKA_EXTERNAL_PORT}
+  #     - KAFKA_CFG_ADVERTISED_LISTENERS=INTERNAL://kafka:${KAFKA_PORT},CLIENT://kafka:${KAFKA_CLIENT_PORT},EXTERNAL://${KAFKA_HOST_NAME}:${KAFKA_EXTERNAL_PORT}
+  #     - KAFKA_CFG_LISTENER_SECURITY_PROTOCOL_MAP=INTERNAL:SSL,EXTERNAL:SSL,CLIENT:SASL_SSL
+  #     - KAFKA_ZOOKEEPER_PROTOCOL=SASL_SSL
+  #     - KAFKA_ZOOKEEPER_USER=${ZOO_SERVER_USERS}
+  #     - KAFKA_ZOOKEEPER_PASSWORD=${ZOO_SERVER_PASSWORDS}
+  #     - KAFKA_ZOOKEEPER_TLS_KEYSTORE_PASSWORD=${ZOO_TLS_CLIENT_KEYSTORE_PASSWORD}
+  #     - KAFKA_ZOOKEEPER_TLS_TRUSTSTORE_PASSWORD=${ZOO_TLS_CLIENT_TRUSTSTORE_PASSWORD}
+  #     - KAFKA_ZOOKEEPER_TLS_VERIFY_HOSTNAME=false
+  #     - KAFKA_CFG_SSL_ENDPOINT_IDENTIFICATION_ALGORITHM=
+  #     - KAFKA_CLIENT_USER=${KAFKA_CLIENT_USER}
+  #     - KAFKA_CLIENT_PASSWORD=${KAFKA_CLIENT_PASSWORD}
+  #     - KAFKA_CERTIFICATE_PASSWORD=${KAFKA_CERTIFICATE_PASSWORD}
+  #     - KAFKA_INTER_BROKER_LISTENER_NAME=INTERNAL
+  #     - KAFKA_CFG_SASL_ENABLED_MECHANISMS=PLAIN
+  #     - KAFKA_CFG_SASL_MECHANISM_INTER_BROKER_PROTOCOL=PLAIN
+  #     - KAFKA_SSL_CLIENT_AUTH=required
+  #   networks:
+  #     - tombolo-network
+  #   volumes:
+  #     - ${KAFKA_ZOOKEEPER_CERTS_PATH}:/bitnami/kafka/config/certs
 
   web:
     build:
       context: ./client-reactjs
       dockerfile: Dockerfile
     networks:
       - tombolo-network
-    environment:
-      PORT: ${WEB_PORT}
-      PROXY_URL: http://node:${PORT}
-      WEB_EXPOSED_PORT: ${WEB_EXPOSED_PORT}
-      WEB_EXPOSED_HTTP_PORT: ${WEB_EXPOSED_HTTP_PORT}
-      HOSTNAME: ${HOST_HOSTNAME}
     ports:
-      - ${WEB_PORT}:${WEB_PORT}
-      - ${WEB_EXPOSED_PORT}:${WEB_EXPOSED_PORT}
-      - ${WEB_EXPOSED_HTTP_PORT}:${WEB_EXPOSED_HTTP_PORT}
+      - ${HTTP_PORT}:80
+      - ${HTTPS_PORT}:443
     volumes:
-      - ${CERT_PATH}:/etc/ssl
+      - ${CERT_PATH}:/etc/nginx/certs/
```

Comment:
Is this directory available on nginx image by default?

Reply:
Yes it is